docs/views/examples/check-your-answers-page.html

```diff
@@ -7,127 +7,163 @@
-      </tr>
-      <tr>
-        <td>
-          Producer
-        </td>
-        <td>
-          Producer name
-        </td>
-        <td class="change-answer">
-          <a href="#">
-            Change <span class="visuallyhidden">Producer name</span>
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td>
-          Site of export
-        </td>
-        <td>
-          Site of export name
-        </td>
-        <td class="change-answer">
-          <a href="#">
-            Change <span class="visuallyhidden">Site of export name</span>
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td>
-          Importer
-        </td>
-        <td>
-          Importer name<br>
-          First line of address<br>
-          Second line of address<br>
-          Contact: Contact Name<br>
-          Tel: 01234 567 890<br>
-          Email: email@email.com
-        </td>
-        <td class="change-answer">
-          <a href="#">
-            Change <span class="visuallyhidden">Importer name</span>
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td>
-          Recovery facilities
-        </td>
-        <td>
-          Recovery facilities name
-        </td>
-        <td class="change-answer">
-          <a href="#">
-            Change <span class="visuallyhidden">Recovery facilities name</span>
-          </a>
-        </td>
-      </tr>
-      <tr>
-        <td>
-          Recovery site
-        </td>
-        <td>
-          Recovery site name
-        </td>
-        <td class="change-answer">
-          <a href="#">
-            Change <span class="visuallyhidden">Recovery site name</span>
-          </a>
-        </td>
-      </tr>
-    </tbody>
-  </table>
-
-  <h2 class="heading-medium">Now send your application</h2>
-
-  <p class="text">
-    By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.
-  </p>
-
-  <div class="form-group">
-    <a href="confirmation-page" class="button">Accept and send application</a>
-  </div>
+      <h1 class="heading-large">
+        Check your answers before sending your application
+      </h1>
+
+
+      <!-- only add a heading for a list if there is more than one list -->
+      <h2 class="heading-medium">
+        Personal details
+      </h2>
+
+      <!-- use additional modifier class to change column widths for multiple groups of questions and answers: -->
+      <!--   * `cya-questions-short` for short questions -->
+      <!--   * `cya-questions-long` for long questions -->
+      <!--   * none for single group of q&a -->
+      <dl class="govuk-check-your-answers cya-questions-short">
+        <div>
+          <dt class="cya-question">
+            Name
+          </dt>
+          <dd class="cya-answer">
+            Sarah Philips
+          </dd>
+          <dd class="cya-change">
+            <a href="#">
+              Change <span class="visuallyhidden">name</span>
```

Comment:
I've got a personal preference for the space after change to be inside the visually hidden span. Feels more *correct* that way.